src/components/EventForm.js

```diff
@@ -0,0 +1,124 @@
+import React from "react";
+import "react-dates/initialize";
+import "react-dates/lib/css/_datepicker.css";
+import { DateRangePicker } from "react-dates";
+import moment from "moment";
+import uuid from "uuid";
+
+export default class EventForm extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      title: "",
+      description: "",
+      organisator: "",
+      city: "",
+      category: "",
+      image: false,
+      id: uuid(),
+      createdAt: moment(),
+      startDate: moment(),
+      startDateId: "",
```

Comment:
How it works right now? I see that you have to use `startDateId` in `DatePicker` component, but how that component handle changes of that value, when you just change the date? Do you even need that prop?

Reply:
I don't know exactly why but startDateId and endDateId in new DateRangePicker are required and without that I wouldn't  be able to make it work.